databrowser/src/locales/en.json

```diff
@@ -40,5 +40,11 @@
       "downloadJSON": "Download dataset as JSON",
       "downloadCSV": "Download dataset as CSV"
     }
+  },
+  "components": {
+    "cancelSave": {
```

Comment:
This seems to me a bit a strange object. Can we remove it and move the two fields just under `components`?

Reply:
Done with 0713945